test/constants.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape');
+
+const constants = require('../constants');
+
+test('Constants', (t) => {
```

Comment:
This is what I wanted to avoid. This level of abstraction is a little unnecessary.